framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -241,6 +246,54 @@ private void executeQuery(String query) throws IOException, SQLException {
     }
   }
 
+  private void executeLimitZeroQuery(String query) throws IOException, SQLException {
+	if (getTestStatus() == TestStatus.CANCELED) {
```

Comment:
The way the test is setup ( testVerifier.verify... throws an exception that is caught), execLimitZeroQuery will be skipped anyway.

> On Dec 2, 2015, at 9:25 PM, Chun Chang <notifications@github.com> wrote:
> 
> In framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java:
> 
> > @@ -241,6 +246,54 @@ private void executeQuery(String query) throws IOException, SQLException {
> >      }
> >    }
> >  
> > +  private void executeLimitZeroQuery(String query) throws IOException, SQLException {
> > +	if (getTestStatus() == TestStatus.CANCELED) {
> There is no harm (in fact preferred) to catch both failures. So we should run limit 0 query. We should not run limit 0 if there is an execution failure. Let me guard on that as well.
> 
> —
> Reply to this email directly or view it on GitHub.
>